app/templates/components/submission-repo-details.hbs

```diff
@@ -0,0 +1,41 @@
+<style>
+  .sub-status { padding-left:23px; }
+  .fas {
+    display: inline;
+    font-size: 0.8em;
+    opacity: 0.9;
+  }
+  .Complete>.fa-circle, .Complete .fa-info-circle {color: #4dbd74;}
+  .stalled>.fa-circle {color: #f86c6b;}
+</style>
+<li>
+  <p class="card-title">
+    <strong>{{repo.name}} ({{#if repo.url}}<a href={{repo.url}}>{{repo.url}}</a>{{/if}})</strong>
+  </p>
+  <p class="card-text">
+    <div class="sub-status">
+      {{#if repoCopy}}
+        {{#if (eq repoCopy.copyStatus "complete")}}
+          <span class="Complete" tooltip-position="left" tooltip="Submission to this repository was accepted and processed that the repository. ID(s) have been assigned to the submitted manuscript">
```

Comment:
Unfortunately there was a typo in the ticket that got carried through. Could also be shortened to say, "Submission was accepted and processed by the repository. ID(s) have been assigned to the submitted manuscript."